commitlint/plugins.ts

```diff
@@ -272,6 +272,28 @@ export abstract class Plugins {
         ];
     }
 
+    public static defaultRevertMessage(headerStr: string, bodyStr: string) {
+        let offence = false;
+
+        if (headerStr.match(/^[Rr]evert ".+"$/) !== null) {
+            // does msg have a body?
+            if (bodyStr !== null) {
+                let lines = bodyStr.split("\n");
+                offence =
+                    // 40 is the length of git commit hash in the following regex pattern.
```

Comment:
@realmarv nit, if you place the comment one line down more (in L284 instead of L283) then you don't need to explain "in the following regex"

Reply:
Done